<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfTrackVisibilityPrefWidget</class>
 <widget class="QWidget" name="WkfTrackVisibilityPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Visibility</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="visibilityGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Track Filter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="showNoTracksRadioButton">
        <property name="text">
         <string>Show No Tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showLocalTracksRadioButton">
        <property name="text">
         <string>Show Only Local Tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showAllTracksRadioButton">
        <property name="text">
         <string>Show All Visible Teams' Tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="teamTrackLayout" stretch="0,0">
        <item>
         <widget class="QRadioButton" name="showTeamTracksRadioButton">
          <property name="text">
           <string>Show Selected Team's Tracks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="showTeamTracksComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="showNetworkedSimulationsTracks">
        <property name="text">
         <string>Show Tracks From Networked Simulations</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="iconAppearanceBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Track Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="mGraphicsVerticalLayout">
      <item>
       <widget class="QCheckBox" name="trackColorCheckbox">
        <property name="text">
         <string>Show Team Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="trackOrientationCheckbox">
        <property name="text">
         <string>Show Track Orientation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="trackLabelCheckbox">
        <property name="text">
         <string>Show Track ID Label</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>54</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>54</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Label Font</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="UtQtFontButton" name="trackFontButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="scaleLayout">
        <item>
         <widget class="QLabel" name="scaleLabel">
          <property name="maximumSize">
           <size>
            <width>54</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Track Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="scaleSlider">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>150</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>150</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="wedgeRadioButton">
        <property name="text">
         <string>Wedge Icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="customIconRadioButton">
        <property name="text">
         <string>Custom Icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="wkf::ImageSelectionWidget" name="customImageSelector" native="true"/>
      </item>
      <item>
       <widget class="QRadioButton" name="rulesRadioButton">
        <property name="text">
         <string>Rules-Based Icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="rulesGroupBox">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="landLabel">
             <property name="text">
              <string>Land:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="wkf::ImageSelectionWidget" name="landImageSelector" native="true"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="airLabel">
             <property name="text">
              <string>Air:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="surfaceLabel">
             <property name="text">
              <string>Surface:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="subsurfaceLabel">
             <property name="text">
              <string>Subsurface:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="spaceLabel">
             <property name="text">
              <string>Space:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="unknownLabel">
             <property name="text">
              <string>Unknown:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="wkf::ImageSelectionWidget" name="airImageSelector" native="true"/>
           </item>
           <item row="2" column="1">
            <widget class="wkf::ImageSelectionWidget" name="surfaceImageSelector" native="true"/>
           </item>
           <item row="3" column="1">
            <widget class="wkf::ImageSelectionWidget" name="subsurfaceImageSelector" native="true"/>
           </item>
           <item row="4" column="1">
            <widget class="wkf::ImageSelectionWidget" name="spaceImageSelector" native="true"/>
           </item>
           <item row="5" column="1">
            <widget class="wkf::ImageSelectionWidget" name="unknownImageSelector" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="platformIconCheckBox">
           <property name="text">
            <string>Use Target Platform Icon</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::ImageSelectionWidget</class>
   <extends>QWidget</extends>
   <header>selectors/WkfImageSelectorWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UtQtFontButton</class>
   <extends>QPushButton</extends>
   <header>UtQtFontButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
